File: systemTests/src/test/java/ServerRegistry.java
import java.util.HashMap;
import java.util.Map;

public class ServerRegistry {
    public static final String BILLING = "billing-test";
    public static final String UMS = "user-management-test";
    public static final String EMAIL = "email-test";

    private static final Map<String, ServerDetails> servers = new HashMap<>();

    public ServerRegistry() {
        servers.put(BILLING, new ServerDetails(BILLING, "localhost:8081", "'counter.billing.reocurringPayment.created'"));
        servers.put(EMAIL, new ServerDetails(EMAIL, "localhost:8080", "'counter.emails.sent'"));
        servers.put(UMS, new ServerDetails(UMS, "localhost:8082", "'counter.ums.subscription.created'"));

    }

    public ServerDetails get(String server) {
        return servers.get(server);
    }
}

File: systemTests/src/test/kotlin/ServerRegistry.kt
import java.util.*

class ServerRegistry {

    init {
        servers.put(BILLING, billingServer)
        servers.put(EMAIL, ServerDetails(EMAIL, "localhost:8080", "'counter.emails.sent'"))
        servers.put(UMS, ServerDetails(UMS, "localhost:8082", "'counter.ums.subscription.created'"))

    }

    operator fun get(server: String): ServerDetails {
        return servers.getOrDefault(server, billingServer)
    }

    companion object {
        val BILLING = "billing-test"
        val UMS = "user-management-test"
        val EMAIL = "email-test"

        private val billingServer = ServerDetails(BILLING, "localhost:8081", "'counter.billing.reocurringPayment.created'")

        private val servers = HashMap<String, ServerDetails>()
    }
}
